src/components/notifications/ui-notification-toast.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useCallback, useEffect, useLayoutEffect, useMemo, useRef, useState } from 'react'
+import { Button, ProgressBar, Toast } from 'react-bootstrap'
+import { UiNotification } from '../../redux/ui-notifications/types'
+import { ForkAwesomeIcon } from '../common/fork-awesome/fork-awesome-icon'
+import { ShowIf } from '../common/show-if/show-if'
+import { IconName } from '../common/fork-awesome/types'
+import { dismissUiNotification } from '../../redux/ui-notifications/methods'
+
+const stepsPerSecond = 10
+
+export interface UiNotificationProps extends UiNotification {
+  notificationId: number
+}
+
+export const UiNotificationToast: React.FC<UiNotificationProps> = ({ title, content, date, icon, dismissed, notificationId, durationInSecond, buttons }) => {
+  const [eta, setETA] = useState<number>()
```

Comment:
When using `eta` in all lowercase, only the E should be upper-case in the setter.

```suggestion
  const [eta, setEta] = useState<number>()
```

Reply:
done